prometheus/src/main/scala/filodb/prometheus/ast/Vectors.scala

```diff
@@ -5,9 +5,6 @@ import filodb.core.query.ColumnFilter
 import filodb.query._
 
 object Vectors {
-  // The "tag" or key used to indicate the column to query to FiloDB.  A non-standard Prom extension.
-  val ColumnSelectorLabel = "__col__"
```

Comment:
This is a non-backward compatible change that needs to be called out during release. It will break any existing clients that use `__col__` notation in their PromQL. Needs to be addressed. (or called out)

Reply:
Yes, it has been called out in the PR notes.